base_frame_id: panda_link0
ee_grasp_offset_z: -0.04

task_frame_id: task
table_height: 0.16  # Measured from the base frame specified above

workspaces:  # Please use max two ;)
# - scan_joints: 
#     - [0.010, -1.479, 0.100, -2.416, 0.089, 1.402, 0.851]
#     - [-0.520, -0.796, 0.912, -1.672, 0.091, 1.281, 0.871]
#     - [0.092, -0.626, -0.551, -1.837, 0.165, 1.238, 0.900]
#   drop_joints: [-0.329, -0.285, -0.289, -1.878, -0.070, 1.635, 0.801]
- scan_joints: 
    - [0.010, -1.479, 0.100, -2.416, 0.089, 1.402, 0.851]
    - [-0.520, -0.796, 0.912, -1.672, 0.091, 1.281, 0.871]
    - [0.092, -0.626, -0.551, -1.837, 0.165, 1.238, 0.900]
  drop_joints: [0.580, -0.256, -0.151, -2.438, -0.025, 2.206, 1.196]
- scan_joints: 
    - [0.010, -1.479, 0.100, -2.416, 0.089, 1.402, 0.851]
    - [-0.520, -0.796, 0.912, -1.672, 0.091, 1.281, 0.871]
    - [0.092, -0.626, -0.551, -1.837, 0.165, 1.238, 0.900]
  drop_joints: [0.062, -0.278, -0.368, -2.50, -0.135, 2.277, 0.5203]

grasp_selection: "auto"  # Choice: manual, auto

arm_velocity_scaling_drop: 0.3
arm_velocity_scaling_grasp: 0.2
